tools/site-creator/create-site.js

```diff
@@ -42,7 +42,6 @@ async function checkCodeBus(data) {
 
 async function previewOrPublishPages(data, action, setStatus) {
   const parent = getDestinationPath(data.repo, data.org);
-  const libraryPath = `${parent}/docs/library`;
```

Comment:
I'm confused - I thought we don't want to publish library either. Why did you revert that?

Reply:
This was your suggested change :D 

https://github.com/hlxsites/aem-boilerplate-commerce/pull/469#discussion_r2098510472

But I though you're moving the library inside the .da folder no?

I think it's best if I just wait with making more changes until you figure out with Markus exactly which paths we need.

Reply:
lmao....ignore me. I should not comment before coffee.

Reply:
And agree - I am also waiting on paths from Markus before I proceed with https://github.com/adobe-commerce/aio-cli-plugin-commerce/pull/26/files